### PR TITLE
Fixes #1357 individual building block for mobi

### LINKED ISSUE #1357: Create Individual Structure (Building block used in MoBi)
OriginData (Should probably be promoted from PKSim to Core)
Cache of Parameters (Path as key, ParameterS)

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockSourcedFromPKSimBuildingBlock<IndividualParameter>
```
Comment (rwmcintosh) [inline src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs:5]: I wasn't sure if CalculationMethodCache should also be part of the MoBi Individual. These are the common building block properties besides that one.

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs:5]: That's a mouthful class name.

File: src/OSPSuite.Core/Domain/Builder/PathAndValueEntityBuildingBlockSourcedFromPKSimBuildingBlock.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public abstract class PathAndValueEntityBuildingBlockSourcedFromPKSimBuildingBlock<T> : PathAndValueEntityBuildingBlock<T> where T : class, IWithPath, IObjectBase
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/Builder/PathAndValueEntityBuildingBlockSourcedFromPKSimBuildingBlock.cs:5]: Hum.....don't like that name

File: src/OSPSuite.Core/Domain/CoreOriginData.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public class CoreOriginData : BaseOriginData
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/CoreOriginData.cs:3]: I don't get the baseOriginData, vs CoreOriginData.
Why do we need this?

Comment (rwmcintosh) [reply src/OSPSuite.Core/Domain/CoreOriginData.cs:3]: The only thing that exists in CoreOriginData is Species as a string. The rest is all the same. In Core we want to be able to set and get that string, but not in PKSim (not as a string anyway)

Comment (rwmcintosh) [reply src/OSPSuite.Core/Domain/CoreOriginData.cs:3]: generic would probably work here. TSpecies or something like that.

File: src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs (hunk 1 above)
Comment (rwmcintosh) [reply src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs:5]: What about PathAndValueEntityBuildingBlockFromPKSim

**Comment by rwmcintosh** [comment]
Removed OriginData from Core. Now we'll just get an enumerable of properties as string that we can display somehow in the view when it is implemented.

File: src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockFromPKSim<IndividualParameter>
+   {
+      public Cache<string, string> OriginData { set; get; } = new Cache<string, string>();
```
Comment (msevestre) [inline src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs:7]: @rwmcintosh As mentiioned in the chat, I think this is not the right abstraction here

**Review by msevestre — APPROVED** [review]

**Pull request merged**